HelpSource/Classes/Server.schelp

```diff
@@ -320,6 +318,15 @@ s.quit;
 method:: peakCPU, avgCPU
 Get peak and average CPU usage.
 
+method:: latency
+The current latency of the server. See link::Guides/ServerTiming:: for details.
+
+method:: sampleRate
+An integer representing the nominal sample rate of the server.
+
+method:: actualSampleRate
+A floating-point number representing the true sample rate of the server.
```

Comment:
The `sampleRate` you give in the server options to specify what nominal rate you want the server to run on. The `actualSampleRate` is the current rate of the hardware, which might drift.